### ISSUE TITLE
parentage of phophosering phosphatase and phosphohistidine phosphatase

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#12183]
neither of these are descendants of 'phosphoprotein phosphatase', so they do not inherit 'protein dephosphorylation'

I'm not sure but I'm not aware of any dephosphorylation of free amino acids, and at least for the higher eukaryotes these look like  protein dephosphorylation.

Images show graph and ALL experimental annotations to these terms

**Comment by ValWood** [comment]
![phosphoserine](https://cloud.githubusercontent.com/assets/7359272/11588111/0690d2bc-9a76-11e5-95c1-24f91263157e.jpg)

**Comment by ValWood** [comment]
![phosphatase](https://cloud.githubusercontent.com/assets/7359272/11588120/1e1029ba-9a76-11e5-8b88-ddc19b74a615.jpg)

**Comment by mah11** [comment]
First of all, these are phosphatase activities, not kinase activities, so they shouldn't have links to phosphorylation of anything. They do have links via the 'phosphatase' parent to DEphosphorylation.

Second, both are defined by their reactions as acting only on free phosphorylated amino acids. I've just learned that phosphoserine phosphatase act late in serine biosynthesis:

http://www.enzyme-database.org/reaction/AminoAcid/Ser.html

So the ontology looks OK.

**Comment by mah11** [comment]
If the annotations are wrong, that should go in a ticket on the annotation tracker.

**Comment by ValWood** [comment]
![annot](https://cloud.githubusercontent.com/assets/7359272/11588390/fdc2cef4-9a77-11e5-86aa-894764b12c14.jpg)

**Comment by ValWood** [comment]
The phosphoserine annotation look OK, there are problems with the phosphohistidine. PHPT1 is a protein phosphatase. I think a "comment' might be helpful on these terms.  The PHPT1 has been used for a LOT of inferred annotations. It might be useful even for the definition to  explicitly state 'free amino acid' too, which should prevent  their use in these cases.

**Comment by ValWood** [comment]
I'll raise a dispute in protein2GO for the  various PHpt1

**Comment by ValWood** [comment]
Actually, I think the histidine phosphatase term is meant to only represent a phosphoprotein phosphatase term....

**Comment by mcourtot** [comment]
Hi Val, I'm away this week but will look into it before the christmas break.

**Comment by mcourtot** [comment]
1. Annotations: I was just checking P2GO and don't see the disputes - am I looking at the wrong place? I couldn't find a ticket in the annotation tracker either.

2. Ontology: Do you think we should add an extra comment to phosphohistidine phosphatase that they act only on free phosphorylated amino acids even though their definition is the explicit reaction? If you think it helps clarify I'm happy doing that.

**Comment by ValWood** [comment]
I didn't raise a dispute because I think all of phosphohistidine phosphatases are "phosphoprotein phosphatases"
If this is the case, this can be made explicit in the def, and the phosphoprotein phosphatase parentage added.

The phophoserine term could be clarified to specify free amino acids.

Val

**Comment by mcourtot** [comment]
I found this paper: http://www.ncbi.nlm.nih.gov/pubmed/25574816, where they seem to indicate that "PHPT1 can dephosphorylate phospholysine in chemically phosphorylated histone H1 and polylysine ", and it may have a broader specificity. What do you think?

**Comment by ValWood** [comment]
I guess it depends if its physiologically relevant and if when the annotators made the existing annotations they were trying to represent "protein phosphorylation", because if they were, they haven't....
I think it would be useful if it were clearer to annotators and there was a parent term which represented  either/or and specific terms to represent   free amino acids vs. peptide phosphorylating version. 

At present if you  are drilling down and you find "phosphohistidine phosphatase" there is no way to clarify which of these you intended.

**Comment by mcourtot** [comment]
I agree and I think the annotators were indeed trying to represent protein phosphorylation (as evidenced by the annotations you highlighted), but I'm wary of modifying this term as this would involve modifying the definition as well in a significant way. @mah11 you initially created this term, what do you think?

Could one solution be to 1. clarify that this term is for free amino acids 2. create a new term under protein phosphatase for "protein histidine phosphatase activity"?

**Comment by mah11** [comment]
I can tell from the ID (GO:0008969) that I did not create the phosphohistidine phosphatase activity term. I don't remember anything much about crafting the definition -- which I did add -- other than that it was a very long time ago. Most likely I modelled it on the analogous definition for phosphoserine phosphatase activity, going essentially entirely by the term names.

I therefore don't have any specific knowledge (if indeed I ever did) that would preclude altering the definition and ancestry, but if you do I would recommend also changing the name to match those of the other phosphatases that act on amino acid residues in proteins, i.e. to 'protein histidine phosphatase activity'.
